exercise1/exercise1.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Exercise 1</title>
+  </head>
+  <style>
+.company_img {
```

Comment:
indentation wrong. This is under style block

Reply:
okay sir